### PR TITLE
Weather app - Mai

### PR DESCRIPTION
**Pull request by maikanetaka**
## Netlify link
https://maik-weather-app.netlify.app/

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+const weatherMessageElement = document.getElementById("weather-message");
+
+// API access and default city configuration
+const BASE_URL = "https://api.openweathermap.org/data/2.5/weather?q=";
+const FORECAST_URL = "https://api.openweathermap.org/data/2.5/forecast?q=";
+const API_KEY = "db63b41efc78e9f2c60ec93f035d8cff";
+const DEFAULT_CITY = "Stockholm,Sweden";
+
+//Error handling
+function handleError(error) {
```
Comment (Caracal23) [inline script.js:18]: Good that you are logging the error.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+function displayForecast(forecastData) {
+  const forecastListElement = document.getElementById("forecast-list");
+  forecastListElement.innerHTML = "";
+  const noonForecasts = forecastData.list.filter((forecast) =>
+    forecast.dt_txt.includes("12:00:00")
+  );
+
+  noonForecasts.forEach((forecast) => {
+    const date = new Date(forecast.dt_txt);
+    const days = ["sun", "mon", "tue", "wed", "thu", "fri", "sat"];
```
Comment (Caracal23) [inline script.js:201]: I like that you approach the days in an array ☺️

File: script.js
Comment (Caracal23) [inline script.js:1]: your code is well structured and clean, which makes it easy to read and understand-

**Review by Caracal23 — APPROVED** [review]

**Comment by Caracal23** [comment]
Great work Mai!!

File: .DS_Store
Comment (AntonellaMorittu) [inline .DS_Store:1]: please delete all the .DS_Store files 🧹

File: script.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+}
+
+// Weather message
+function displayWeatherMessage(condition, cityName, weather) {
+  let detailedMessage;
+  let weatherIcon;
+  let weatherDescription = weather.description;
+  console.log(
+    condition
+  )
```
Comment (AntonellaMorittu) [inline script.js:147]: please delete 🧹

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Mai, very good job this week! Your code is well structured and formatted. The app is working as it should, you completed the basic requirements to pass this assignment. Is the search bar not working? Then please delete if not fully functional 👍  One comment is about the basic requirements: 

- The app should have: 4-day forecast You are displaying all 5 days when we requested 4 
- the temperature should be rounded to 1 decimal 

Not a big deal, fix these things and you're good to go:) Keep up the awesome work 🎸

**Comment by maikanetaka** [comment]
Hello! @AntonellaMorittu, I have fixed to be 4 days forecast and 1 decimal. Please check!

**Comment by AntonellaMorittu** [comment]
Hey Mai! Did you deploy using another link or is the one submitted valid? I can't see the changes, please double-check, thank you

**Comment by maikanetaka** [comment]
Hi @AntonellaMorittu Sorry for that! I manually deployed and now it should work.  https://maik-weather-app.netlify.app/ please check!

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hey Mai, great almost there, still missing:
- the temperature should be rounded to 1 decimal

**Comment by maikanetaka** [comment]
Hi @AntonellaMorittu, I have changed to 1 decimal. Please check.

**Review by AntonellaMorittu — APPROVED** [review]

**Pull request closed without merging**